src/noteServices/addNoteService.js

```diff
@@ -1,14 +1,14 @@
-import { token } from "../token";
 import axios from "axios";
 export const AddNoteHandler = async (data) => {
+  const authToken = localStorage.getItem("authToken")
   try {
     const res = await axios.post(
       "/api/notes",
       {
         note: data 
       },
       {
-        headers: { authorization: token }
+        headers: { authorization: authToken }
```

Comment:
Use of token for authentication is correctly  done.

Reply:
Thank you!